utils/xmtpRN/contentTypes/index.ts

```diff
@@ -26,7 +26,10 @@ export function isContentType(args: {
   return contentType.startsWith(prefix);
 }
 
-export function getMessageContentType(contentType: string) {
+export function getMessageContentType(contentType: string | undefined) {
+  if (!contentType) {
+    return undefined;
+  }
```

Comment:
I wonder if we should put | undefined here 🤔 

And instead handle the undefined before calling the function.

Curious on your thoughts. It's just that I feel it's counterintuitive to call this function with a contentType that is undefined haha.

Love the `| undefined` instead of `?: string` tho! At least it shows it wants an argument